x-pack/plugins/infra/public/pages/metrics/metric_detail/index.tsx

```diff
@@ -19,11 +21,25 @@ export const NodeDetail = () => {
     params: { type: nodeType, node: nodeName },
   } = useRouteMatch<{ type: InventoryItemType; node: string }>();
 
-  useMetricsBreadcrumbs([
-    {
-      text: nodeName,
-    },
-  ]);
+  const breadCrumbs: ChromeBreadcrumb[] = [];
+
+  const hostsLinkProps = useLinkProps({
+    app: 'metrics',
+    pathname: 'hosts',
+  });
+
+  if (nodeType === 'host') {
```

Comment:
In this case, I think we need to check where the user is coming from - you can reach the host details page when you navigate from the inventory page for example (by clicking on a host and then clicking on open as page): 
<img width="1894" alt="image" src="https://github.com/elastic/kibana/assets/14139027/f18d5263-814e-4cbd-ab17-7628fe43f82d">
So here we should have [Observability] > [Infrastructure] > [Inventory] > Host Name. The node type is still `host` in this example. I would suggest using the history to check where the user is coming from (we can use the `useHistory` hook and check `history.location.state.originPathname`) -  That way it can be dynamic based on the previous user location.
In case the user opens this page directly without any `history` we can maybe set `Hosts` for the case the `nodeType` is `host` and `Inventory` when it's not. Wdyt?